classMemory.js

```diff
@@ -0,0 +1,26 @@
+(function() {
+    "use strict";
+
+    function Memory() {
+        this.num = 0;
+    }
+
+    Memory.prototype.mPlus = function(number) {
+        this.num += Number(number);
+    }
+
+    Memory.prototype.mMinus = function(number) {
+        this.num += Number(-number);
```

Comment:
You should not converting anything to number here -- it is a caller's responsibility (the one who is using this method) to provide number as a parameter.